doc/examples/reconst_fwdti.py

```diff
@@ -0,0 +1,174 @@
+"""
+=====================================================================
+Title
```

Comment:
Yeah. This should probably be labeled as "WIP", huh? :-)

It still needs to be rebased on top of https://github.com/nipy/dipy/pull/1036, which still needs to be merged, so I imagine this will change quite a bit.